packages/protocol/contracts/governance/LockedGold.sol

```diff
@@ -81,7 +81,7 @@ contract LockedGold is
   * @return The storage, major, minor, and patch version of the contract.
   */
   function getVersionNumber() external pure returns (uint256, uint256, uint256, uint256) {
-    return (1, 1, 1, 2);
+    return (1, 1, 1, 3);
```

Comment:
just bumped up the patch version as i only added a transparent view function, curious i was thinking about this correctly

Reply:
The versioning scheme is describe here: https://docs.celo.org/community/release-process/smart-contracts#versioning. So I think you will actually need to bump to 1.1.2.0. `protocol-test-compatibility`, once it runs (not sure why CI is stalled on this PR), should fail and tell you that this needs to be updated.